packages/json/src/JsonEditorField.tsx

```diff
@@ -61,7 +59,6 @@ export function JsonEditorField(props: JsonEditorFieldProps) {
         value={props.value}
         onChange={props.onChange}
         theme="light"
-        extensions={[json(), EditorView.lineWrapping]}
```

Comment:
What are the consequences of this? Is it fair to expect support tickets regarding it?

Reply:
we are going to test the difference this makes before merging, it looks more like a styling thing at first glance but once we confirm we'll know the potential impact